fetch.js

```diff
@@ -246,14 +246,20 @@ function Body() {
       this._bodyText = body = Object.prototype.toString.call(body)
```

Comment:
Alternatively, we can set `this._bodyInit` to equal `this._bodyText` in this closure. I'm not sure about the implications of doing this, so would appreciate a feedback.

Reply:
whiteslack